Proxy/HTTP/FilterAuthenticatedNonBearerTokens.bambda

```diff
@@ -0,0 +1,56 @@
+/**
+ * Filter when an Authorization header is present, not empty and does not include a traditional bearer token (beginning with "ey")
+ *
+ * @author GangGreenTemperTatum (https://github.com/GangGreenTemperTatum)
+ **/
+
+var configNoFilter = true;
+var configNotInScopeOnly = true;
+var sessionCookieName = "";
+var sessionCookieValue = "";
+
+if (!requestResponse.hasResponse()) {
+    return false;
+}
+
+var request = requestResponse.request();
+var response = requestResponse.response();
+
+if (!response.isStatusCodeClass(StatusCodeClass.CLASS_2XX_SUCCESS)) {
+    return false;
+}
+
+var authHeader = request.hasHeader("Authorization");
```

Comment:
if `hasHeader()` doesn't find a result, it will return null, so you do not need to use the ternary below.